### ISSUE TITLE
PieChart invalidateData (strange effect)

### ISSUE THREAD

**Issue by ptorrent** [amcharts/amcharts4#472]
Hello,

Example here :

https://codepen.io/ptorrent/pen/JmvZGQ

```javascript
function change(){
  chart.data[2].litres = Math.random() * 10200
  chart.invalidateData()
  setTimeout(change,1000)
}

setTimeout(change,1000)
```

**Comment by martynasma** [comment]
Help me out: what is strange about that demo? Could not manage to understand :)

**Comment by ptorrent** [comment]
Appears on Chrome, Opera

![amchart](https://user-images.githubusercontent.com/6083644/47219662-affb0000-d3af-11e8-8ce4-7c1cc7991325.gif)

**Comment by martynasma** [comment]
Gotcha. Thanks.

Looks like for a split second a pie is drawn before the legend, causing the jump.

We'll need to look into it.

**Comment by martynasma** [comment]
Fixed in [4.0.0-beta.63](https://github.com/amcharts/amcharts4/releases/tag/4.0.0-beta.63).

## [4.0.0-beta.63] - 2018-10-20

### Fixed
- In some cases a `PieChart` could overlap its legend.
- `PieChart` flicker after `invalidateData` fixed.
- An error was occurring when data was updated on chart with series with bullet while hovering it with cursor.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
